source/includes/getting-started/_install.html.md

```diff
@@ -0,0 +1,265 @@
+# Install Prerequisites
+
+[//]: # (If any commented statements become false, change prior text)
+
+Before you begin, you will need to have access to our Beta. If you don't have access yet, request access at [bons.ai][1] and then come visit us in the [Forums][17] and let us know why you're excited to try out Bonsai!
```

Comment:
Does it make sense to have a link to the forums here or is the left nav enough? Sounds kind of arrogant to me to say "come and tell us how excited you are to use our product" might want to change that to something more humble.